src/components/commons/ChainStatsCard.tsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import clsx from "clsx";
+import BigNumber from "bignumber.js";
+
+interface StatsCardProps {
+  cardTitle: string;
```

Comment:
let's remove `card..` for the three props naming here, if we have two title maybe, then adding a differentiator will help, but right now this is not the case